modules/rostests/apitests/ntdll/locale.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * PROJECT:         ReactOS api tests
+ * LICENSE:         GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
+ * PURPOSE:         RTL locale support
+ * PROGRAMMER:      Pierre Schweitzer <pierre@reactos.org>
+ */
```

Comment:
Is it Pierre's code? Probably need a correct copyright year.
```suggestion
/*
 * PROJECT:     ReactOS API tests
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later.html)
 * PURPOSE:     RTL locale support
 * COPYRIGHT:   Copyright 2021 Pierre Schweitzer <pierre@reactos.org>
 */
```

Reply:
Yes. I just moved it to a new file in order to be able to use it elsewhere. I'll have a look at the commit history for the year.